### PR TITLE
initial commit for importc.d

### PR DESCRIPTION
**Pull request by WalterBright**
This is the boilerplate for adding the documentation file for ImportC.

### CODE REVIEW

File: spec/spec.dd [hunk 1]
```diff
@@ -53,6 +53,7 @@ $(TOC Table of Contents,
                 $(A abi.html, Application Binary Interface),
                 $(A simd.html, Vector Extensions),
                 $(A betterc.html, Better C),
+                $(A importc.html, Importc C),
```
Comment (CyberShadow) [inline spec/spec.dd:56]: I'm guessing you meant
```suggestion
                $(A importc.html, ImportC),
```

Comment (WalterBright) [reply spec/spec.dd]: fixed

**Comment by WalterBright** [comment]
ping @CyberShadow

File: spec/importc.dd [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Ddoc
+
+$(SPEC_S ImportC,
+
+$(HEADERNAV_TOC)
+
+$(P Documentation for ImportC coming soon!)
+
+$(SPEC_SUBNAV_PREV_NEXT betterC, Better C, ob, Live Functions)
```
Comment (CyberShadow) [inline spec/importc.dd:9]: This results in a broken link on case sensitive filesystems, I think the correct line would be:
```suggestion
$(SPEC_SUBNAV_PREV_NEXT betterc, Better C, ob, Live Functions)
```

Comment (WalterBright) [reply spec/importc.dd]: that's why I'm doing this boilerplate stuff as a separate PR! Done.

**Review by CyberShadow — APPROVED** [review]

**Pull request merged**